.github/workflows/push-build.yaml

```
@@ -37,16 +40,24 @@ jobs:
           aws-region: us-east-1
 
       - name: unshallow
-        run: git fetch --prune --unshallow --tags
+        run: git fetch --prune --unshallow --tags --force
 
       # creating custom env var
       - name: set env
-        run: echo "BRANCH_NAME=${GITHUB_REF##*/}" >> $GITHUB_ENV
+        run: |
+          echo "BRANCH_NAME=${GITHUB_REF##*/}" >> $GITHUB_ENV
+          echo "GITHUB_REF"=${GITHUB_REF} >> $GITHUB_ENV
 
       - name: build and release
         env:
           GIT_API_TOKEN: ${{ secrets.GIT_API_TOKEN }}
           BRANCH_NAME: ${{ env.BRANCH_NAME }}
           AWS_USR: ${{ secrets.AWS_USR }}
           AWS_PSW: ${{ secrets.AWS_PSW }}
-        run: tests/scripts/build-release.sh
+          GITHUB_REF: $ {{ env.GITHUB_REF }}
+        run: |
+          if [[ ${GITHUB_REF} =~ master|v* ]]; then
```

Do we need more comparison for tags than "v*"? And seems like we don't need the "*" since it's already a regular expression comparison?
```suggestion
          if [[ ${GITHUB_REF} =~ master|tags/v ]]; then
```

---

removed `*` but why do we need for `tags` as a check-in if condition as we put our tags with `v*`?